core/storage/topic/gae_models.py

```diff
@@ -165,6 +167,95 @@ class TopicSummaryModel(base_models.BaseModel):
     version = ndb.IntegerProperty(required=True)
 
 
+class SubtopicPageSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
+    """Storage model for the metadata for a subtopic page snapshot."""
+    pass
+
+
+class SubtopicPageSnapshotContentModel(base_models.BaseSnapshotContentModel):
+    """Storage model for the content of a subtopic page snapshot."""
+    pass
+
+
+class SubtopicPageModel(base_models.VersionedModel):
+    """Model for storing Subtopic pages.
+
+    This stores the HTML data for a subtopic page.
+    """
+    SNAPSHOT_METADATA_CLASS = SubtopicPageSnapshotMetadataModel
+    SNAPSHOT_CONTENT_CLASS = SubtopicPageSnapshotContentModel
+    ALLOW_REVERT = False
+
+    # The topic id that this subtopic is a part of.
+    topic_id = ndb.StringProperty(required=True, indexed=True)
+    # The html data of the subtopic.
+    html_data = ndb.TextProperty(required=True)
+    # The ISO 639-1 code for the language this topic is written in.
```

Comment:
should be "subtopic page" rather than "topic", right?

Reply:
Yeah, done.